tap_logmeinrescue/streams/base.py

```diff
@@ -161,6 +162,9 @@ def sync_data(self, parent_ids):
 
         start_date = get_last_record_value_for_table(
             self.state, table, 'start_date')
+        if utils.strptime_to_utc(start_date) > utils.now():
+            start_date == utils.strftime(utils.now())
```

Comment:
So start_date is the bookmark above and there's a chance its greater than `now()` which is right-this-second?